Index: lvgl_ili9341-master/main.c
===================================================================
--- lvgl_ili9341-master.orig/main.c
+++ lvgl_ili9341-master/main.c
@@ -29,8 +29,8 @@ int main(void)
     lv_disp_drv_init(&disp_drv);
     disp_drv.draw_buf   = &disp_buf;
     disp_drv.flush_cb   = fbdev_flush;
-    disp_drv.hor_res    = 800;
-    disp_drv.ver_res    = 480;
+    disp_drv.hor_res    = 320;
+    disp_drv.ver_res    = 240;
     lv_disp_drv_register(&disp_drv);
 
     evdev_init();
