Index: palera1n-main/Makefile
===================================================================
--- palera1n-main.orig/Makefile
+++ palera1n-main/Makefile
@@ -1,13 +1,13 @@
 NAME = bakepale
 
 SRC = $(shell pwd)
-DEP = /usr/local/lib
+DEP = $(STAGING_DIR)/usr/lib
 # CC ?= gcc
-CFLAGS += -I$(SRC)/include -I$(SRC)
+CFLAGS += -I$(SRC)/include -I$(SRC) -I$(STAGING_DIR)/usr/include
 CFLAGS += -Wall -Wextra -DPALERAIN_VERSION=\"2.0.0\" -Wall -Wextra -Wno-unused-parameter
 CFLAGS += -Wno-unused-variable -I$(SRC)/src -std=c99 -pedantic-errors -D_C99_SOURCE -D_POSIX_C_SOURCE=200112L
-LIBS += -L$(DEP) -limobiledevice-1.0 -lirecovery-1.0 -lusbmuxd-2.0
-LIBS += -limobiledevice-glue-1.0 -lplist-2.0 -lssl -lcrypto -lm -lrt -ldl -lpthread
+LIBS += -limobiledevice-1.0 -lirecovery-1.0 -lusbmuxd-2.0
+LIBS += -lplist-2.0 -lssl -lcrypto -lm -lrt -ldl -lpthread
 ifeq ($(TARGET_OS),)
 TARGET_OS = $(shell uname -s)
 endif
@@ -72,7 +72,8 @@ palera1n: download-deps
 	$(MAKE) -C src
 
 install:
-	cp src/palera1n /usr/sbin
+	echo "test"
+	# cp src/palera1n /usr/sbin
 
 clean:
 	$(MAKE) -C src clean
