Index: rtl8723ds-sdio-master/core/rtw_bt_mp.c
===================================================================
--- rtl8723ds-sdio-master.orig/core/rtw_bt_mp.c
+++ rtl8723ds-sdio-master/core/rtw_bt_mp.c
@@ -20,6 +20,7 @@
 
 
 #include <drv_types.h>
+#include <rtw_mp.h>
 #include <rtw_bt_mp.h>
 
 #if defined(CONFIG_RTL8723B)
Index: rtl8723ds-sdio-master/hal/phydm/rtl8723d/halphyrf_8723d.h
===================================================================
--- rtl8723ds-sdio-master.orig/hal/phydm/rtl8723d/halphyrf_8723d.h
+++ rtl8723ds-sdio-master/hal/phydm/rtl8723d/halphyrf_8723d.h
@@ -21,6 +21,8 @@
 #ifndef __HAL_PHY_RF_8723D_H__
 #define __HAL_PHY_RF_8723D_H__
 
+#include <rtw_mp.h>
+
 /*--------------------------Define Parameters-------------------------------*/
 #define	IQK_DELAY_TIME_8723D		10		//ms
 #define	index_mapping_NUM_8723D	15
Index: rtl8723ds-sdio-master/hal/phydm/rtl8723d/halhwimg8723d_fw.h
===================================================================
--- rtl8723ds-sdio-master.orig/hal/phydm/rtl8723d/halhwimg8723d_fw.h
+++ rtl8723ds-sdio-master/hal/phydm/rtl8723d/halhwimg8723d_fw.h
@@ -23,6 +23,7 @@
 #ifndef __INC_MP_FW_HW_IMG_8723D_H
 #define __INC_MP_FW_HW_IMG_8723D_H
 
+#include <rtw_mp.h>
 
 /******************************************************************************
 *                           FW_AP.TXT
Index: rtl8723ds-sdio-master/hal/phydm/rtl8723d/phydm_regconfig8723d.h
===================================================================
--- rtl8723ds-sdio-master.orig/hal/phydm/rtl8723d/phydm_regconfig8723d.h
+++ rtl8723ds-sdio-master/hal/phydm/rtl8723d/phydm_regconfig8723d.h
@@ -19,7 +19,9 @@
  ******************************************************************************/
 #ifndef __INC_ODM_REGCONFIG_H_8723D
 #define __INC_ODM_REGCONFIG_H_8723D
- 
+
+#include <rtw_mp.h>
+
 #if (RTL8723D_SUPPORT == 1)
 
 void
Index: rtl8723ds-sdio-master/hal/phydm/rtl8723d/phydm_rtl8723d.h
===================================================================
--- rtl8723ds-sdio-master.orig/hal/phydm/rtl8723d/phydm_rtl8723d.h
+++ rtl8723ds-sdio-master/hal/phydm/rtl8723d/phydm_rtl8723d.h
@@ -20,6 +20,8 @@
 #ifndef	__ODM_RTL8723D_H__
 #define __ODM_RTL8723D_H__
 
+#include <rtw_mp.h>
+
 #if (RTL8723D_SUPPORT == 1)
 
 s1Byte 
Index: rtl8723ds-sdio-master/hal/phydm/phydm_debug.h
===================================================================
--- rtl8723ds-sdio-master.orig/hal/phydm/phydm_debug.h
+++ rtl8723ds-sdio-master/hal/phydm/phydm_debug.h
@@ -22,6 +22,8 @@
 #ifndef	__ODM_DBG_H__
 #define __ODM_DBG_H__
 
+#include <rtw_mp.h>
+
 /*#define DEBUG_VERSION	"1.1"*/  /*2015.07.29 YuChen*/
 /*#define DEBUG_VERSION	"1.2"*/  /*2015.08.28 Dino*/
 #define DEBUG_VERSION	"1.3"  /*2016.04.28 YuChen*/
Index: rtl8723ds-sdio-master/hal/phydm/phydm_interface.h
===================================================================
--- rtl8723ds-sdio-master.orig/hal/phydm/phydm_interface.h
+++ rtl8723ds-sdio-master/hal/phydm/phydm_interface.h
@@ -22,6 +22,8 @@
 #ifndef	__ODM_INTERFACE_H__
 #define __ODM_INTERFACE_H__
 
+#include <rtw_mp.h>
+
 #define INTERFACE_VERSION	"1.1"		/*2015.07.29  YuChen*/
 
 //
Index: rtl8723ds-sdio-master/hal/phydm/rtl8723d/halhwimg8723d_bb.h
===================================================================
--- rtl8723ds-sdio-master.orig/hal/phydm/rtl8723d/halhwimg8723d_bb.h
+++ rtl8723ds-sdio-master/hal/phydm/rtl8723d/halhwimg8723d_bb.h
@@ -23,6 +23,7 @@
 #ifndef __INC_MP_BB_HW_IMG_8723D_H
 #define __INC_MP_BB_HW_IMG_8723D_H
 
+#include <rtw_mp.h>
 
 /******************************************************************************
 *                           AGC_TAB.TXT
Index: rtl8723ds-sdio-master/hal/phydm/rtl8723d/halhwimg8723d_mac.h
===================================================================
--- rtl8723ds-sdio-master.orig/hal/phydm/rtl8723d/halhwimg8723d_mac.h
+++ rtl8723ds-sdio-master/hal/phydm/rtl8723d/halhwimg8723d_mac.h
@@ -23,6 +23,7 @@
 #ifndef __INC_MP_MAC_HW_IMG_8723D_H
 #define __INC_MP_MAC_HW_IMG_8723D_H
 
+#include <rtw_mp.h>
 
 /******************************************************************************
 *                           MAC_REG.TXT
Index: rtl8723ds-sdio-master/hal/phydm/rtl8723d/halhwimg8723d_rf.h
===================================================================
--- rtl8723ds-sdio-master.orig/hal/phydm/rtl8723d/halhwimg8723d_rf.h
+++ rtl8723ds-sdio-master/hal/phydm/rtl8723d/halhwimg8723d_rf.h
@@ -23,6 +23,7 @@
 #ifndef __INC_MP_RF_HW_IMG_8723D_H
 #define __INC_MP_RF_HW_IMG_8723D_H
 
+#include <rtw_mp.h>
 
 /******************************************************************************
 *                           RadioA.TXT
Index: rtl8723ds-sdio-master/hal/phydm/phydm.h
===================================================================
--- rtl8723ds-sdio-master.orig/hal/phydm/phydm.h
+++ rtl8723ds-sdio-master/hal/phydm/phydm.h
@@ -25,6 +25,7 @@
 /*============================================================*/
 /*include files*/
 /*============================================================*/
+#include <rtw_mp.h>
 #include "phydm_pre_define.h"
 #include "phydm_dig.h"
 #include "phydm_edcaturbocheck.h"
Index: rtl8723ds-sdio-master/hal/phydm/phydm_hwconfig.h
===================================================================
--- rtl8723ds-sdio-master.orig/hal/phydm/phydm_hwconfig.h
+++ rtl8723ds-sdio-master/hal/phydm/phydm_hwconfig.h
@@ -22,6 +22,7 @@
 #ifndef	__HALHWOUTSRC_H__
 #define __HALHWOUTSRC_H__
 
+#include <rtw_mp.h>
 
 /*--------------------------Define -------------------------------------------*/ 
 #define	CCK_RSSI_INIT_COUNT 5
