Index: u-boot-2022.07/arch/arm/dts/Makefile
===================================================================
--- u-boot-2022.07.orig/arch/arm/dts/Makefile
+++ u-boot-2022.07/arch/arm/dts/Makefile
@@ -168,6 +168,7 @@ dtb-$(CONFIG_ROCKCHIP_RK3399) += \
 	rk3399pro-rock-pi-n10.dtb
 
 dtb-$(CONFIG_ROCKCHIP_RK3568) += \
+	rk3566-cputools-i6p.dtb \
 	rk3568-evb.dtb \
 	rk3566-quartz64-a.dtb \
 	rk3568-rock-3a.dtb
